Week3/homework/2-step3.js

```diff
@@ -1,8 +1,21 @@
 'use strict';
 
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
-  const values = [];
-  // Add your code here
+    const values = [];
+    // Add your code here
+    for (let i = startIndex; i <= stopIndex; i++) {
+        values.push(i);
+        if (i % 3 === 0) sayThree(i);
+        if (i % 5 === 0) sayFive(i);
+    }
 }
 
 threeFive(10, 15, sayThree, sayFive);
+
+function sayThree(number) {
+    console.log(number + ' is divisible by three');
+}
+
+function sayFive(number) {
+    console.log(number + ' is divisible by five');
+}
```

Comment:
✅ Nicely done.  Very concise, and you only run through once, adding to the array as you go.  Very nice.